automated install/basic-install.sh

```diff
@@ -21,6 +21,10 @@
 # instead of continuing the installation with something broken
 set -e
 
+# Set PATH to a usual default to assure that all basic commands are available.
+# When using "su" an uncomplete PATH could be passed: https://github.com/pi-hole/pi-hole/issues/3209
+export PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
```

Comment:
This will kill any user configured PATH's. If you want to modify PATH (and I don't suggest we do this) then append to existing PATH, don't overwrite the entire PATH.

Reply:
> This will kill any user configured PATH's.

But all essential commands are available via default paths anyway? And note that when using `sudo` (which the installer does itself when being executed as non-root) and in some other cases any custom PATH is lost anyway and since a script is a non-interactive bash session even `.bashrc` and such are not sourced.

But yes adding is an alternative, but due to required `:` is not existing, it would look a bid ugly:
```
if [[ $PATH ]]; then
  export PATH+=':/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
else
  export PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
fi
```

A look to add single entries if not yet contained would be cleanest but too large coding effort IMO 😉.

I'd go with killing custom path, which can be even seen as security/consistency step to assure that default commands are used. But if you like the above alternative more, I can replace it. However decide first whether you want to touch this at all to fix such rare failures as linked.

Reply:
Or `export PATH=$PATH:/whatever`?

Or my preferred option, tell users to not run `su` to install.

Reply:
Ah I just checked and leading `:` is not an issue, i.e. when PATH was empty before, so no if-then-else required:
```
export PATH+=':/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
```
or
```
export PATH="$PATH:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```
both results are exactly the same, but first one slightly shorter.

Okay will adjust it to preserve custom entries. I don't like the fact that in 99% of cases one will simply have an exact doubled PATH with this, but practically nothing changes besides the hopefully rare `su` users.

> Or my preferred option, tell users to not run su to install.

I totally agree, am big fans of `sudo` instead for several reasons, but on Debian `su` is essentially integrated while `sudo` is a package with priority "optional"...